clang/test/CIR/CodeGen/atomic.cpp

```diff
@@ -1271,3 +1271,75 @@ void lock_test_and_set(unsigned short* a, short b) {
 void lock_test_and_set(unsigned char* a, char b) {
   unsigned char c = __sync_lock_test_and_set(a, b);
 }
+
+// CHECK-LABEL: @_Z4swapPii
+// CHECK: cir.atomic.fetch(xchg, {{.*}} : !cir.ptr<!s32i>, {{.*}} : !s32i, seq_cst) fetch_first : !s32i
+
+// LLVM-LABEL: @_Z4swapPii
+// LLVM: atomicrmw xchg ptr {{.*}}, i32 {{.*}} seq_cst, align 4
```

Comment:
Nit: please add `OGCG` lines too (see `clang/test/CIR/CodeGen/alignment.c` for examples). No need to add it for the existing tests, just the new ones are fine.